core-developers/memorialization.rst

```diff
@@ -0,0 +1,146 @@
+
+Rationale
+=========
+
+When a core developer passes away, memorializing accounts helps create
+a space for remembering the contributor and protects against attempted
+logins and fraudulent activity.
+
+The process
+===========
+
+The memorialization process is performed by a member of the PSF staff
+with administrative access to current and historical systems where
+core developers have access.
+
+After the status of the core developer in question is confirmed,
+access to the systems listed below is revoked and some changes are
+made to how the user displays to others.
+
+To respect the choices that someone made while alive, we aim to preserve
+content of their accounts without changes after they've passed away.
+To support the bereaved, in some instances, we may remove or change
+certain content when the legacy contact or family members request it.
+
+GitHub
+------
+
+* The user is removed from the `python/ <https://github.com/orgs/python/>`_
+  organization on GitHub;
+* The user is removed from the `psf/ <https://github.com/orgs/psf/>`_
+  organization on GitHub;
+* The user is removed from the `pypa/ <https://github.com/orgs/pypa/>`_
+  organization on GitHub.
+
+The PSF staff does not follow up with GitHub with regards to GitHub account
+cancellation as this action is reserved for next-of-kin or designated by
+the deceased GitHub user to act as an account successor.
+
+The general policy regarding deceased users on GitHub is described
+`here <https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy>`_.
+
+Repositories in the organization
+--------------------------------
+
+* The user's GitHub handle is removed from ``/.github/CODEOWNERS``.
+  To see all that need action, perform
+  `this query <https://github.com/search?q=org%3Apython+path%3A**%2F.github%2FCODEOWNERS+USERNAME&type=code>`_.
+* The user is marked as deceased in the private
+  `voters/python-core.toml <https://github.com/python/voters/blob/main/python-core.toml>`_
+  file with the ``left=`` field set to the day of passing, if known.
+
+discuss.python.org
+------------------
+
+* The user's "custom status" is set to 🕊 ``in memoriam``;
+* The user's "about me" is amended with ``$firstname passed away on $date. [In memoriam.]($in_memoriam_post_url)``;
+* In the user's security "recently used devices" the staff member chooses "Log out all";
+* In the user's permissions the staff member chooses "Deactivate account";
+* The user's trust level is reset to ``1: basic user`` (trust level 0 doesn't allow links in "About Me");
+* The user's "associated accounts" (like GitHub) that provide an alternative
+  login method, are all disconnected;
+* The user's API keys are revoked;
+* The user's admin or moderator right is revoked;
+* The user's primary email address is reset to ``USERNAME@in-memoriam.invalid`` and
+  secondary email addresses are removed (this step requires the administrator
+  to contact Discourse.org staff via ``team@discourse.org``)
```

Comment:
Minor nit, this list item is missing the `.`.

Reply:
Fixed in #1517.